src/box/txn_limbo.c

```diff
@@ -52,13 +52,15 @@ txn_limbo_create(struct txn_limbo *limbo)
 	limbo->rollback_count = 0;
 	limbo->is_in_rollback = false;
 	limbo->svp_confirmed_lsn = -1;
+	limbo->frozen = false;
 }
 
 bool
 txn_limbo_is_ro(struct txn_limbo *limbo)
 {
-	return limbo->owner_id != REPLICA_ID_NIL &&
-	       limbo->owner_id != instance_id;
+	return limbo->frozen ||
```

Comment:
That change will also block asynchronous and fully local transactions even if the limbo is empty but the quorum is lost. Was it intentional? Looks dubious TBH.

Reply:
Making limbo RO while it's frozen was proposed by @sergepetrenko [original comment](https://github.com/tarantool/tarantool/pull/6981#discussion_r853119679). IMO we should block async transactions in case there is a synchronous one waiting in frozen limbo and the asynchronous one is likely to be rolledback after replicasets connectivity is restored, because it's likely that there is a new leader elected. But I'm open to discuss this decision.

Reply:
> That change will also block asynchronous and fully local transactions even if the limbo is empty but the quorum is lost. Was it intentional? Looks dubious TBH.

But the transactions will be blocked anyway because raft stepped down.
Btw, it won't affect fully local transactions.

Do you think that disabling raft on a fenced node should make it writeable?  This sounds strange IMO.

Reply:
Still looks really fragile to me. In case of a good enough but swiftly blinking network we will often get errors about the instance being read-only. Perhaps the fencing should have waited for synchro timeout before resign. Would have the same effect (no rollbacks nor confirms until quorum is here), but would be more robust against almost unnoticeable network issues.
Anyway, I think if that will happen, we will see complaints and will be able to fix it.